reference/conanfile/attributes.rst

```diff
@@ -924,52 +924,49 @@ This object should be filled in ``package_info()`` method.
-executable will be one component inside ``cpp_info`` like this (the following case is not a real example):
+:ref:`Using components <package_information_components>` you can achieve a more fine-grained control over individual libraries available in
+a single Conan package. Components allow you define a ``cpp_info`` like object per each of those libraries and also requirements between
+them and to components of other packages (the following case is not a real example):
 
 .. code-block:: python
 
     def package_info(self):
         self.cpp_info.name = "OpenSSL"
-        self.cpp_info.components["crypto"].name = "Crypto"
+        self.cpp_info.components["crypto"].names["cmake_find_package"] = "Crypto"
         self.cpp_info.components["crypto"].libs = ["libcrypto"]
-        self.cpp_info.components["ssl"].name = "SSL"
+        self.cpp_info.components["crypto"].defines = ["DEFINE_CRYPTO=1"]
+        self.cpp_info.components["crypto"].requires = ["zlib::zlib"]  # Depends on all components in zlib package
+
+        self.cpp_info.components["ssl"].names["cmake"] = "SSL"
+        self.cpp_info.components["ssl"].includedirs = ["include/headers_ssl"]
         self.cpp_info.components["ssl"].libs = ["libssl"]
+        self.cpp_info.components["ssl"].requires = ["crypto",
+                                                    "boost::headers"]  # Depends on headers component in boost package
 
-You can also model system dependencies for each component or just header files.
+The interface of the ``Component`` object is the same as the one used by the ``cpp_info`` object (except for the ``requires`` attribute) and
+has **the same default directories**.
 
 .. warning::
 
     Using components and global ``cpp_info`` non-default values or release/debug configurations at the same time is not allowed (except for
     ``self.cpp_info.name`` and ``self.cpp_info.names``).
 
+Dependencies among different components can be defined using the ``requires`` attribute and the name of the component. The dependency graph
+for components will be calculated and values will be aggregated in the correct order for each field.
+
 .. important::
 
     Components information is still not available from the consumer side (``self.deps_cpp_info`` doesn't provide the ``components``
-    dictionary). We are planning to complete this feature in next releases.
-
-    The information of components is not lost but aggregated to the *global* scope and the usage of components should be transparent right
-    now to consumers and generators.
-
-The structure of the ``Component`` object is the same as the one used by the ``cpp_info`` object and has **the same default directories**.
+    dictionary). The information of components is not lost but aggregated to the *global* scope and the usage of components should be
+    transparent right now to consumers and generators.
```

Comment:
I'd remove this `important` and add a link to the `package_information_components`. Take into account the `seealso` note below, probable both of them can be joined.